docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -118,7 +111,16 @@ Any numerical attribute on the span is fair game.
 { name = "GET /:endpoint" } | min_over_time(span.http.status_code)
 ```
 
+## The `max_over_time` function
```

Comment:
Possibly could just combine with the above just like you did with the quantile_over_time and histogram_over_time functions. Alternatively if you want to keep them separate and you think my suggestions above are helpful, you can transfer them here. If not, no worries. :)

Reply:
I'll combine them like we did the other section.

Reply:
Done.